kivymd/uix/refreshlayout/refreshlayout.py

```diff
@@ -175,8 +177,26 @@ class MDScrollViewRefreshLayout(MDScrollView):
     and defaults to `None`.
     """
 
+    spinner_color = ColorProperty()
```

Comment:
spinner_color = ColorProperty(None)

Reply:
Do I have to reassign it if I'm absolutely satisfied with default value `[1, 1, 1, 1]` ?